systemtests/src/test/java/io/enmasse/systemtest/bases/web/ConsoleTest.java

```diff
@@ -1419,6 +1424,134 @@ protected void doTestErrorDialog(AddressSpace addressSpace) throws Exception {
 
     }
 
+    protected void doTestOpenShiftWithCustomCert() throws Exception {
+        String openshiftConfigNamespace = "openshift-config";
+        String openshiftIngressNamespace = "openshift-ingress";
+        String openshiftIngressOperatorNamespace = "openshift-ingress-operator";
+        String openshiftAuthenticationNamespace = "openshift-authentication";
+
+        Map<String, String> oauthDeploymentLabels = Map.of(LabelKeys.APP, "oauth-openshift");
+        Map<String, String> oauthRouteLabels = Map.of(LabelKeys.APP, "oauth-openshift");
+        Map<String, String> consoleDeploymentLabels = Map.of(LabelKeys.APP, "enmasse", LabelKeys.NAME, "console");
+
+        Optional<Deployment> oauth = kubernetes.listDeployments(openshiftAuthenticationNamespace, oauthDeploymentLabels).stream().findFirst();
+        assertThat(oauth.isPresent(), is(true));
+
+        Optional<Route> oauthRoute = kubernetes.listRoutes(openshiftAuthenticationNamespace, oauthRouteLabels).stream().findFirst();
+        assertThat(oauthRoute.isPresent(), is(true));
+
+        Optional<Deployment> console = kubernetes.listDeployments(consoleDeploymentLabels).stream().findFirst();
+        assertThat(console.isPresent(), is(true));
+
+        CertPair originalOauthCert = OpenSSLUtil.downloadCert(oauthRoute.get().getSpec().getHost(), 443);
+
+        String customCaCnName = "custom-ca";
+        String customIngressSecretName = "custom-ingress";
+        try (CertPair ca = OpenSSLUtil.createSelfSignedCert("/O=io.enmasse/CN=MyCA");
+             CertPair unsignedCluster = OpenSSLUtil.createSelfSignedCert("/O=io.enmasse//CN=MyCluster");
+             CertSigningRequest csr = OpenSSLUtil.createCsr(unsignedCluster);
+             CertPair cluster = OpenSSLUtil.signCrs(csr, Collections.singletonList("*.apps-crc.testing"), ca)
```

Comment:
this is really CRC specific, it cause issues when we will run it agains real cluster, better is use `Environment.getKubernetesDomain()`

Reply:
Fixed. I've now run the test on a proper OpenShift 4 cluster.  It passes.